metricbeat/module/couchdb/_meta/kibana/7/dashboard/Metricbeat-couchdb-overview.json

```
@@ -0,0 +1,1339 @@
+              "i": "4",
+              "w": 24,
+              "x": 24,
+              "y": 0
+            },
+            "panelIndex": "4",
+            "panelRefName": "panel_3",
+            "version": "7.3.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 10,
+              "i": "5",
+              "w": 24,
+              "x": 24,
+              "y": 25
+            },
+            "panelIndex": "5",
+            "panelRefName": "panel_4",
+            "version": "7.3.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 10,
+              "i": "6",
+              "w": 16,
+              "x": 8,
+              "y": 0
+            },
+            "panelIndex": "6",
+            "panelRefName": "panel_5",
+            "version": "7.3.0"
+          },
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 10,
+              "i": "7",
+              "w": 24,
+              "x": 0,
+              "y": 25
+            },
+            "panelIndex": "7",
+            "panelRefName": "panel_6",
+            "version": "7.3.0"
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Metricbeat CouchDB] Database Overview",
+        "version": 1
+      },
+      "id": "a3ab9a60-b952-11e9-a579-f5c0a5d81340",
+      "migrationVersion": {
+        "dashboard": "7.3.0"
+      },
+      "references": [
+        {
+          "id": "8e15ece0-b94e-11e9-a579-f5c0a5d81340",
+          "name": "panel_0",
+          "type": "visualization"
+        },
+        {
+          "id": "1b60bb70-b94f-11e9-a579-f5c0a5d81340",
+          "name": "panel_1",
+          "type": "visualization"
+        },
+        {
+          "id": "f8c29f10-b94f-11e9-a579-f5c0a5d81340",
+          "name": "panel_2",
+          "type": "visualization"
+        },
+        {
+          "id": "702584a0-b950-11e9-a579-f5c0a5d81340",
+          "name": "panel_3",
+          "type": "visualization"
+        },
+        {
+          "id": "abfaf130-b951-11e9-a579-f5c0a5d81340",
+          "name": "panel_4",
+          "type": "visualization"
+        },
+        {
+          "id": "06018310-b952-11e9-a579-f5c0a5d81340",
+          "name": "panel_5",
+          "type": "visualization"
+        },
+        {
+          "id": "496910f0-b952-11e9-a579-f5c0a5d81340",
+          "name": "panel_6",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2019-08-08T10:47:05.116Z",
+      "version": "Wzg3MiwxXQ=="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "[Metricbeat CouchDB] HTTP Status Codes",
```

Visualization titles should have the module tag at the end
```suggestion
        "title": "HTTP Status Codes [Metricbeat CouchDB]",
```

And to make them nicer, once placed in the dashboard, you can edit and select different titles there.

---

done